packages/components/src/radio/tests/chromatic/RadioGroup.stories.tsx

```diff
@@ -13,7 +13,7 @@ import { RadioGroup } from "../../src/RadioGroup.tsx";
 import { RadioList } from "../../src/RadioList.tsx";
 
 const meta = {
-    title: "Components/Radio/RadioGroup",
+    title: "Components/RadioGroup",
```

Comment:
What if we put all form elements under a **Form** category like RAC does?

```suggestion
    title: "Components/Form/RadioGroup",
```

Reply:
alright, I'm ok with this. so it would be for both chromatic and docs? (but I guess docs will eventually not exist in storybook)